### PR TITLE
Features/amal

### PR DESCRIPTION
**Pull request by amalkahsay**
-done about page 
-changed the text colour on the admin page 
-started working on the security and Privacy pages + Q&A

### CODE REVIEW

File: src/views/About.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+</template>
+
+<script>
+import PrismicItems from '@/components/prismic/PrismicItems'
+export default {
+  name: 'about',
+  components: {
+    PrismicItems
+  },
+  props: ['about'],
```
Comment (simonesquad) [inline src/views/About.vue:32]: These props should be 'content' to match the computed and how it is called in the component above.

File: src/views/About.vue [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      </a>
+       </div></div></section>
+  </section>
+</div>
+</template>
+
+<script>
+import PrismicItems from '@/components/prismic/PrismicItems'
+export default {
+  name: 'about',
```
Comment (simonesquad) [inline src/views/About.vue:28]: This is fine as just the name of the component.

File: src/views/About.vue [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  },
+  props: ['about'],
+  data () {
+    return {
+      response: null
+    }
+  },
+  computed: {
+    content () {
+      const content = this.$store.getters['contentStore/getAbout']
```
Comment (simonesquad) [inline src/views/About.vue:40]: The only thing I would say here is that I believe there is KEY already made in APP_CONSTANTS for this.

Comment (simonesquad) [reply src/views/About.vue:40]: @amalkahsay

**Comment by amalkahsay** [comment]
Q&A/Terms and services pages are running, need to add more content an… …
0a8a006
…d link them to each other

File: src/router/index.ts [hunk 4]
```diff
@@ -40,6 +47,46 @@ const routes: Array<RouteConfig> = [
     name: 'homepage',
     components: { default: Home, header: MainNavbar, footer: MainFooter }
   },
+  {
+    path: '/all',
```
Comment (ThomasLefebvreCS) [inline src/router/index.ts:51]: I think this (and all the following path) are wrong : indeed, they refer to some under pages of the /my-items page. Then their path shouldn't just be /all, /minted etc.
Here, it created empty useless pages. You can see them by going on http://localhost:8086/all.

File: src/views/Security.vue [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+     </b-row>
+   </b-container>
+    </section>
+</div>
+</template>
+
+<script>
+import PrismicItems from '@/components/prismic/PrismicItems'
+export default {
+  name: 'Security',
```
Comment (ThomasLefebvreCS) [inline src/views/Security.vue:61]: Why is it called security if it is the Term and conditions page ?

File: src/views/QAndA.vue [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+     </b-row>
+   </b-container>
+  </section>
+</div>
+</template>
+
+<script>
+import PrismicItems from '@/components/prismic/PrismicItems'
+export default {
+  name: 'qanda',
```
Comment (ThomasLefebvreCS) [inline src/views/QAndA.vue:28]: Should be called faq

File: src/views/About.vue [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+                    <prismic-items :prismicItems="content.text1"></prismic-items>
+          </div>
+        </b-col>
+      </b-row>
+    </b-container>
+    <section data-v-7ba5bd90="" id="getInTouchSection" class="d-flex align-items-center flex-column my-5" style="min-height: 100vh;">
+      <div class="my-auto text-center text-white"><h2 class="text-white"></h2><div class="my-4"></div><div class="mb-3">
+      </div><div class="mb-4 blurb"><span> <prismic-items :prismicItems="content.title2"></prismic-items><prismic-items :prismicItems="content.text2"></prismic-items>
+      </span></div><div class="m-5 d-flex justify-content-center">
+      <a title="join us on discord" href="https://discord.com/invite/sQaKVft" class="mr-4"><b-button variant="light">Button</b-button>
```
Comment (ThomasLefebvreCS) [inline src/views/About.vue:19]: Why is this discord link called "Button" ? :smile:

**Review by ThomasLefebvreCS — CHANGES_REQUESTED** [review]
Great progress Amal, still a few things to change : 
- Some pages don't have a good name (QandA -> faq, security -> Terms and conditions)
- The paths /all, /minted etc already exist, so you don't have to add them to the router. They are part of the /my-items page.
- Some minor design/UI problems still exist

**Pull request closed without merging**